src/terms/bitvector.ml

```diff
@@ -655,7 +641,7 @@ let pp_print_signed_machine_integer ppf b =
                | 16 -> bv16_to_num b
                | 32 -> bv32_to_num b
                | 64 -> bv64_to_num b
```

Comment:
Eliminate the explicit match cases as you did for `pp_print_unsigned_machine_integer`.